patch/Support-for-fullcone-nat.patch

```diff
@@ -1,6 +1,7 @@
 From 67cc2bd46c7ad1431ddd1819cdfad6ecc08a7593 Mon Sep 17 00:00:00 2001
+
```

Comment:
I’d leave it how `git format-patch` generates it.

Reply:
all patches are generated by stg